### PR TITLE
Create neural_network_ploting_with_bp_algorithm_and_without_algorithm

### PR DESCRIPTION
**Pull request by Chloe3013**
initial commit

### CODE REVIEW

File: neural_network_ploting_with_bp_algorithm_and_without_algorithm [hunk 1]
```diff
@@ -0,0 +1,457 @@
+#Neural Network Ploting with BP Algorithm and without Algorithm
```
Comment (ytyky) [inline neural_network_ploting_with_bp_algorithm_and_without_algorithm:1]: Hello, the first line should start with a single hashtag #, followed by a single whitespace, and then the title.

Comment (Chloe3013) [reply neural_network_ploting_with_bp_algorithm_and_without_algorithm]: Thank you. I have solved this problem.

**Review by ytyky — COMMENTED** [review]
additionally, is neural_network_ploting_with_bp_algorithm_and_without_algorithm a Rmd file? You probably need to append .Rmd at the end of your file name

**Comment by ytyky** [comment]
looks great! thank you for your contribution 🎉

**Pull request merged**